ui.tests/test-module/specs/aemEmbedContainer/aemEmbedContainer.runtime.spec.js

```diff
@@ -125,4 +125,81 @@ describe("Sites with Aem Embed Container", () => {
             })
         })
     })
+
+    context('embed multiple form in site ', function () {
```

Comment:
Please create a separate spec for this. This is not part of aem embed container.

Reply:
done